packages/core/src/services/datasetsV2/createFromFile.ts

```diff
@@ -38,34 +36,15 @@ export const createDatasetFromFile = async (
   const diskResult = await disk.putFile(key, data.file)
   if (diskResult.error) return diskResult
 
-  const readCsvResult = await syncReadCsv(data.file, {
-    delimiter: data.csvDelimiter,
-  })
-
-  if (readCsvResult.error) return readCsvResult
-
-  const uniqueHeaders = new Set(readCsvResult.value.headers)
-
-  if (uniqueHeaders.size === 0) {
-    return Result.error(new BadRequestError('CSV file must contain headers'))
-  }
-
-  if ([...uniqueHeaders].some((h) => h.length === 0)) {
-    return Result.error(
-      new BadRequestError('CSV header cannot be an empty string'),
-    )
-  }
-
-  let columns: Column[] = []
-  const newColumns = Array.from(uniqueHeaders.keys()).map((columnName) => ({
-    name: columnName,
-  }))
-  columns = buildColumns({
-    newColumns,
-    prevColumns: columns,
```

Comment:
QA that uploading a CSV in new datasets works. I just moved this to a common function because I use in 2 places. But it should work as before.

Reply:
done